PPA builds failing

RUN: /usr/share/launchpad-buildd/bin/in-target unpack-chroot --backend=chroot --series=bionic --arch=ppc64el

```plain
const void*, size_t)’, declared with attribute warn_unused_result [-Werror=unused-result]
         write(fd, any_mime_data, sizeof any_mime_data);
         ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
~~~~
RUN: /usr/share/launchpad-buildd/bin/in-target unpack-chroot --backend=chroot --series=bionic --arch=amd64

```plain
/<<BUILDDIR>>/wlcs-1.0.0+dev52-g8ad6488/tests/primary_selection.cpp:151:14: error: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Werror=unused-result]
         write(fd, any_mime_data, sizeof any_mime_data);
         ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Etc

--- a/tests/primary_selection.cpp
+++ b/tests/primary_selection.cpp
@@ -148,7 +148,7 @@ struct StubPrimarySelectionSourceListener : PrimarySelectionSourceListener
 
     void send(zwp_primary_selection_source_v1*, const char*, int32_t fd)
     {
-        write(fd, any_mime_data, sizeof any_mime_data);
+        ASSERT_THAT(write(fd, any_mime_data, sizeof any_mime_data), Eq(sizeof any_mime_data));
         close(fd);
     }
 };

--- a/tests/primary_selection.cpp
+++ b/tests/primary_selection.cpp
@@ -148,7 +148,7 @@ struct StubPrimarySelectionSourceListener : PrimarySelectionSourceListener
 
     void send(zwp_primary_selection_source_v1*, const char*, int32_t fd)
     {
-        ASSERT_THAT(write(fd, any_mime_data, sizeof any_mime_data), Eq(sizeof any_mime_data));
+        ASSERT_THAT(write(fd, any_mime_data, sizeof any_mime_data), Eq(ssize_t(sizeof any_mime_data)));
         close(fd);
     }
 };

